relay-event-normalization/src/normalize/mod.rs

```diff
@@ -44,23 +45,33 @@ mod request;
 mod stacktrace;
 
 /// Defines a builtin measurement.
-#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[derive(Debug, Default, Clone, Serialize, Deserialize, PartialEq, Hash, Eq)]
```

Comment:
Do we need this `PartialEq`?

Reply:
im using it for unit test as well, and also for the partialeq of globalconfig